_posts/2019-11-14-automating-next-generation-911-law-enforcement-boundaries.md

```diff
@@ -0,0 +1,39 @@
+
+In order to facilitate Utah's transition to the NG911 system, AGRC is preparing statewide GIS data sets that will drive the call-routing and agency recommendations. Although the transition is still many months away, we are already working on preparing the GIS data. The emergency service boundaries will need to be accurate, with no gaps or overlaps, to ensure that every location has appropriate and unambiguous responding agency recommendations. For additional information on AGRC's NG911 efforts, read [Utah's NG911 GIS Roadmap]({{site.baseurl}}{% post_url 2018-08-31-utah-ng911-gis-roadmap-v1 %}).
+
+## Python Script
+
+With that information in mind, let's get back to AGRC's Python script. Our new script will automatically build jurisdiction polygons based on other boundaries that are maintained in the SGID (e.g., counties, municipalities). Police departments generally cover their own municipal boundaries, while county sheriff's offices generally cover the rest of the county land that doesn't belong to a municipality. However, not all municipalities have their own police department, and some police departments cover multiple municipalities. Further, there are unique law enforcement agencies that aren't associated with either county or municipal boundaries (e.g., Ute tribal police). Each of these nuances is handled by the Python script, which references external files that list which police departments cover which municipalities or contain static boundaries for the unique law enforcement agencies. ![NG911 Law Enforcement Boundaries]({{ "/images/201911_NG911LawEnforcementBoundaries.jpg" | prepend: site.baseurl }})
+
+This script will allow the NG911 boundaries to be dynamically updated as boundaries are adjusted, new municipalities form, or annexations occur. The basic script logic is as follows:
+
+1. Generate sheriff's office jurisdictions from SGID county boundaries
+2. Generate police department jurisdictions from SGID municipal boundaries
+   - Only build boundaries for municipalities that have their own police department
+   - Merge boundaries of municipalities that share a police department
+3. Insert police department boundaries into sheriff's office boundaries to create a combined boundaries layer
+4. Insert unique jurisdiction boundaries into combined boundaries layer
+
+A web map of the current law enforcement boundaries (November 2019) can be viewed [here](http://utah.maps.arcgis.com/home/webmap/viewer.html?useExisting=1). Please explore this map and let us know if there are any errors or discrepancies in your area. We want to ensure that these boundaries are as accurate as possible, and we welcome feedback that leads to better data.
```

Comment:
we tend to avoid here links
```suggestion
If you would like to see the current law enforcement boundaries (November 2019) browse to this [web map](http://utah.maps.arcgis.com/home/webmap/viewer.html?useExisting=1). Please explore this map and let us know if there are any errors or discrepancies in your area. We want to ensure that these boundaries are as accurate as possible, and we welcome feedback that leads to better data.
```

Reply:
(Lol I couldn't think of how to reword this to avoid the "here." Glad you fixed it!)